next-gen/src/OpenTelemetry.AutoInstrumentation.Sdk/OpenTelemetryProtocol/Tracing/ProtobufOtlpSpanExporterAsync.cs

```diff
@@ -0,0 +1,378 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Concurrent;
+
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.OpenTelemetryProtocol.Serializer;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Tracing;
+
+namespace OpenTelemetry.OpenTelemetryProtocol.Tracing;
+
+/// <summary>
+/// OTLP span exporter.
+/// </summary>
+internal sealed class ProtobufOtlpSpanExporterAsync : ProtobufOtlpExporterAsync<SpanBatchWriter>, ISpanExporterAsync
```

Comment:
Seeing this new class hierarchy being added is interesting.

Why not just do this...

```diff
internal sealed class OtlpSpanExporterAsync
   : OtlpExporterAsync<
-     OtlpCollectorTrace.ExportTraceServiceRequest,
+     byte[]
      SpanBatchWriter>, ISpanExporterAsync
```

The whole point of the generic request type was to make it easy to switch from the proto objects to `byte[]` when manual serialization was added 😄